src/buttons/buttons.js

```diff
@@ -24,6 +24,10 @@ angular.module('ui.bootstrap.buttons', [])
 
       //ui->model
       element.bind(buttonsCtrl.toggleEvent, function () {
+        if ('disabled' in attrs) {
+          return;
```

Comment:
Is this the preferred syntax over if(attrs.disabled) which dropwdownToggle and tab directives use?